cli/tests/pcluster/models/cluster_dummy_model.py

```diff
@@ -178,6 +183,10 @@ def dummy_awsbatch_cluster_config(mocker):
             S3Access("dummy-readwrite-bucket", type="READ_WRITE"),
         ]
     )
+    cluster.monitoring = Monitoring(
```

Comment:
note: CW dashboard should be enabled by default sho you don't need to add this section.